test/unit/pipeline/FetchUnitTest.cc

```diff
@@ -429,7 +430,7 @@ TEST_P(PipelineFetchUnitTest, idleLoopBufferDueToNotTakenBoundary) {
   // Fetch the next block of instructions from memory and change the expected
   // outcome of the branch predictor
   fetchUnit.requestFromPC();
-  EXPECT_CALL(predictor, predict(_, _, _))
```

Comment:
I think we should be doing `predict(_, _, _, true)` given we are in a loop and expect the value to be true.

Reply:
See above comment